crates/op/host-op-system/src/lib.rs

```diff
@@ -26,7 +26,16 @@ wasmtime::component::bindgen!({
 });
 
 pub struct SysCtx {
-    // TODO: add more fields
+    sys: sysinfo::System,
+}
+
+impl Default for SysCtx {
+    fn default() -> Self {
+        let mut sys = sysinfo::System::new_all();
+        std::thread::sleep(sysinfo::MINIMUM_CPU_UPDATE_INTERVAL);
```

Comment:
It is not appropriate to sleep here. Let the user to define the interval of how long to measure. For example, get_processes() takes an argument to specify the time interval and this method can go to sleep.

Reply:
This call happens before any wasm component being executed and runs only once, the real interval of measurements is controlled by the interval of get-processes calls in wasm component.

This sleep here is to gurantee the accuracy of the first get-processes call.